products/healthcare/api.yaml

```diff
@@ -424,9 +424,21 @@ objects:
               - parser_config.0.allow_null_header
               - parser_config.0.segment_terminator
               - parser_config.0.schema
+              - parser_config.0.version
             description: |
               JSON encoded string for schemas used to parse messages in this
               store if schematized parsing is desired.
+          - !ruby/object:Api::Type::Enum
+            name: version
+            description: |
+              The version of the unschematized parser to be used when a custom `schema` is not set.
+            exact_version: beta
```

Comment:
We usually prefer min_version that way the private provider will also have all beta fields. If this field is in beta but not alpha, can you add a comment pointing that out?

Reply:
Changed to min.